### ISSUE TITLE
error when start rails server

### ISSUE THREAD

**Issue by Asnadatta** [pitr/angular-rails-templates#120]
I am getting this error when I try to start my rails sever.
```
/home/asnad/.rvm/gems/ruby-2.2.0/gems/angular-rails-templates-0.1.4/lib/angular-rails-templates/engine.rb:49:in `block (2 levels) in <class:Engine>': undefined method `register_engine' for nil:NilClass (NoMethodError)
	from /home/asnad/.rvm/gems/ruby-2.2.0/gems/angular-rails-templates-0.1.4/lib/angular-rails-templates/engine.rb:39:in `each'
	from /home/asnad/.rvm/gems/ruby-2.2.0/gems/angular-rails-templates-0.1.4/lib/angular-rails-templates/engine.rb:39:in `block in <class:Engine>
```
but when I remove gem 'angular-rails-templates' from my gem file it starts again today is my first day on angular :-) I am using rails 4.2
Any help?

**Comment by LookAtMyGit** [comment]
Same here

**Comment by LookAtMyGit** [comment]
Hey, @Asnadatta 
I've just solved it by using older sprockets version.
Add to your Gemfile:
``gem 'sprockets', '2.12.3'``
And run in terminal:
``bundle update``

**Comment by michaelstokes** [comment]
Thank you @LookAtMyGit 

After installing, It must have been my sprockets that had updated to a later version and keeping with 

```
gem 'sprockets', '2.12.3'
```

Thank you.

**Comment by chrisvel** [comment]
+1 here

**Comment by Asnadatta** [comment]
thanks @LookAtMyGit.

**Comment by catuss-a** [comment]
+1 here, fresh rails 4 app + angular-rails-template 0.1.4 raise this exception when launching the server. Downgrading sprockets fixed it.

**Comment by LiMiTz2k** [comment]
Adding sprockets removes the error, but my templates don't load up from the  <ui-view> tag in the application.html.erb file.

**Comment by alereisan** [comment]
Same like @LiMiTz2k, templates dont show within ui-view...

**Comment by novarac23** [comment]
+1 for this :+1: 

Is anybody planning to make `angular-rails-templates` compatible with newer versions of sprockets like `~>3.0` ?

**Comment by pitr** [comment]
Fixed by #126. Version 1.0.0.beta1 is out, I'll release version 1.0.0 once we are stable.
